demo/backend/navigation/navigator/bottom-tab/tab-2.xml.njk

```diff
@@ -16,17 +20,17 @@ hv_button_behavior: "back"
       target="tab-1"
```

Comment:
switch to href

Reply:
[bdba9be](https://github.com/Instawork/hyperview/pull/969/commits/bdba9be7457df7c236cfc745aa08ef9f318dbeb8)